lib/assessment.js

```diff
@@ -81,9 +80,10 @@ module.exports = {
     try {
       assessment_text_templated = ejs.render(assessment.text, context);
     } catch (e) {
-      return ERR(e, callback);
+      debug('renderText()', 'error when rendering text of assessment ', assessment.id, assessment);
+      assessment_text_templated = null;
```

Comment:
Why are we catching this error? I think this should continue to fail fast like the original implementation.